Python/PythonWithCpp/Ctypes/MathCalc/CMakeLists.txt

```
@@ -0,0 +1,20 @@
+cmake_minimum_required(VERSION 3.12.0)
+project(MathCalc)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall")
+
+if (UNIX)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wextra")
```

If you want to add some custom compile options you can use 
```suggestion
target_compile_options(${CMAKE_PROJECT_NAME} PRIVATE -Wall -Wextra and so on)
```